trustymail/trustymail.py

```diff
@@ -197,7 +196,15 @@ def spf_scan(domain):
                 result = "neutral"
 
             try:
-                query = spf.query("127.0.0.1", "email_wizard@" + domain.domain_name, domain.domain_name, strict=2)
+                # Here I am using the IP address for c1b1.ncats.cyber.dhs.gov
+                # (64.69.57.18) since it (1) has a valid PTR record and (2) is
+                # not listed by anyone as a valid mail server.
+                #
+                # I'm actually temporarily using an IP that
+                # virginia.edu resolves to until we resolve why Google
+                # DNS does not return the same PTR records as the CAL
+                # DNS does for 64.69.57.18.
+                query = spf.query("128.143.22.36", "email_wizard@" + domain.domain_name, domain.domain_name, strict=2)
```

Comment:
I'm interested in the long-term solution to this, given @dhs-ncats' intent to craft tools to be reusable outside of our specific needs. I wonder if setting an IP-with-a-valid-PTR-record-that-doesn't-send-mail needs to be something a user supplies at the command line or as an environmental variable in order to run.

Reply:
Other folks can use our IP-with-a-valid-PTR-record-that-doesn't-send-mail once the wrinkles are ironed out with our DNS.  The IP I was going to use does have a valid PTR record (although it isn't publicly available yet) and does not send mail.

Having such an IP as a precondition for using trustymail seems to put a lot of burden on the user.  I don't personally have such an IP, for example, because I don't own any static IPs.  I would have to find one.